src/components/map/MapService.js

```diff
@@ -779,7 +779,7 @@ goog.require('ga_urlutils_service');
           var dsP = Cesium.KmlDataSource.load(config3d.url, {
             camera: scene.camera,
             canvas: scene.canvas,
-            proxy: new Cesium.DefaultProxy(gaGlobalOptions.ogcproxyUrl)
+            proxy: new Cesium.DefaultProxy(gaGlobalOptions.proxyUrl + 'http/')
```

Comment:
That might be an issue in fact.
DefaultProxy expects a full url? Here we use http per default in the background.